test/constants.js

```diff
@@ -15,6 +15,7 @@ export const SERVER_PORTS = Object.freeze({
   httpPort: SERVER_PORT_START,
   httpsPort: SERVER_PORT_START + 1,
   apiPort: SERVER_PORT_START + 2,
+  apiProtocol: 'https',
```

Comment:
@de-laz Out of curiosity, why is the protocol defined here when it's already defined in the `/config/test.json`?

Reply:
Also, I don't think it should be kept within the server ports object since it's an independent concept?

Reply:
@MattyJ007 I have made the changes at https://github.com/jembi/openhim-core-js/pull/1073/commits/4fa66a83dbf4fbb77d0f0dd73adc2a29dbd59c9b